_inc/client/my-plan/my-plan-body.jsx

```diff
@@ -680,33 +680,36 @@ class MyPlanBody extends React.Component {
 								</h3>
 								<p>
 									{ __(
-										'Need help? Search our support site to find out about your site, your account, and how to make the most of WordPress.'
+										'Need help? Learn about getting started, customizing your site, using advanced code snippets and more.'
```

Comment:
Shouldn't there be an Oxford comma here?

```suggestion
										'Need help? Learn about getting started, customizing your site, using advanced code snippets, and more.'
```

Reply:
Good catch, thank you!